libs/blocks/global-navigation/features/search/gnav-search.js

```diff
@@ -122,12 +119,13 @@ class Search {
     });
   }
 
-  getSuggestions(query = this.query) {
+  async getSuggestions(query = this.query) {
     const { env } = getConfig();
+    const { search } = await getServiceConfig(window.location.origin);
```

Comment:
would this have any impact on performance?

Reply:
Not really, because `getServiceConfig` uses cached data, and the `origin` is accessed through the `window` object. So, overall, I think there is no impact on performance.